supertokens_python/utils.py

```diff
@@ -78,6 +78,11 @@ def find_max_version(
     return max_v
 
 
+def is_version_lt(version: str, minimum_minor_version: str) -> bool:
+    assert len(minimum_minor_version.split(".")) == 2
+    return get_max_version(version, minimum_minor_version) != version
+
+
 def get_max_version(v1: str, v2: str) -> str:
```

Comment:
Perhaps you should rename this to `_get_max_version` so that we know not to use this function anymore. Cause it's still possible that someone in the future might end up incorrectly using `get_max_version`.